Saule/Serialization/ApiError.cs

```diff
@@ -50,6 +73,26 @@ private static string GetRecursiveExceptionMessage(HttpError ex)
             }
 
             return msg;
+        }
+
+        private ExceptionRenderer GetRenderer()
```

Comment:
I imagine one would need too be supplied at runtime, otherwise, people would need to define their own GetRenderer